docs/upgrade-guide/upgrade-6.9.0-7.0.0.md

```diff
@@ -0,0 +1,185 @@
+---
+Title: Upgrading from ADF v6.9 to v7.0
+---
+
+# Upgrading from ADF v6.9 to v7.0
+
+This guide provides instructions on how to upgrade your v6.9.0 ADF projects to v7.0.0.
+
+## Before you begin
+
+Always perform upgrades on "clean" project state, backup your changes or make a project backup.
+
+```shell
+# Recommended clean up
+nx reset && rm -rf .angular .nx dist node_modules nxcache tmp
+```
+
+## Libraries updates
+
+Node version used is now 20.18.1.
+
+### Breaking changes with libraries:
+
+- Angular: 17.1.3
+- Angular Material: 17.1.2
+- Typescript: 5.3.3
+- RXJS: 7.8.1
+- NX: 20.0.0
+
+Angular updates can be done with "Update Guide" from Angular documentation.
+
+### Added libraries:
+- eslint-plugin-storybook
+- jasmine-marbles
+- @editorjs/paragraph
+- editorjs-text-alignment-blocktune
+- graphql-ws
+
+### Deleted libraries:
+- protractor
+- selenium-webdriver
+- webdriver-manager
+- shx
+- monaco-editor
+- ngx-monaco-editor-v2
+- @types/selenium-webdriver
+- protractor-retry-angular-cli
+- protractor-screenshoter-plugin
+- protractor-smartrunner
+
+Reinstall your dependencies and make initial build:
+
+```shell
+npm i --legacy-peer-deps
+npm run build
+```
+
+Review your applications as some styles and classes of Angular Material components might have changed.
+
+
+## Demo-Shell and e2e
+Demo shell and its e2e tests have been deleted. Protractor e2e tests have been refactored to Playwright.
```

Comment:
I'm wondering on last part of this point. It's mentioned that e2es are refactored to Playwright but I guess in ADF we don't have e2es at all actually? We have playwright in dependency probably but it's unused (we don't have e2es now because we don't have demo shell anymore). Could you please check it with some QA and correct this text? Also we may need to raise task to remove playwright from dependency in ADF if it's unused (please check with QA)